Added Stefan Riedmann

StefanRiedmann opened the pull request:
If you are submitting a new blog please read:

**Please do not forget to add the cs file to the csproj file!**

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, if HTTPS is used, the certificate is valid
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [x] I have applied a filter (Optional: if blog is only about Xamarin)
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

StefanRiedmann commented:
removed csproj attributes and added https support to my blog

Cheesebaron commented on src/Firehose.Web/Firehose.Web.csproj:
```diff
@@ -16,10 +16,6 @@
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <UseIISExpress>true</UseIISExpress>
     <IISExpressSSLPort>44371</IISExpressSSLPort>
-    <IISExpressAnonymousAuthentication />
```
Please don't change any of these settings

StefanRiedmann replied:
I'll add this again - a mistake when reverting the changes that VS made on compiling/testing the app.

Cheesebaron replied:
It is not only this line, which you added back, it is the whole blog. Check your commits.

StefanRiedmann commented:
I just recently added https support. I'll have to refactor the internal image references, to avoid mixed content.

StefanRiedmann commented:
Now it should be fine, hopefully. I removed this lines, probably because of a misunderstood change request from my first fork (last week), that I already deleted.

Cheesebaron commented:
Looks good to me now. Welcome to Planet Xamarin!

A little advice on future posts
- Don't post code as images, it looks horrible on mobile devices and doesn't scale with screen DPI. There are excellent code block plugins for WP which you seem to be using.
- Those collapsing sections hide the contents of the post, which annoys me. Why do I have to click to read content? Why does it collapse other sections when I click? I can't cross reference what was written in a previous section if it gets closed.
- Some of your posts say "x comments" where x is a number, then going to that post, no comments are shown in the discuss part on the bottom.

The content seems pretty good, I've read some of your blog posts, so keep up the work with adding new content, but at least consider the points above.

StefanRiedmann commented:
Thank you very much for your feedback!

The 'x comments' problem has to do with Wordpress, I still couldnt figure it out. Somehow it regards cross-references between posts as comments.

I think I'll remove these sections - I'm just a bit obsessed sometimes with seeing the structure of a document when reading it...

And I'll definitly look into finding a good code plugin. Until now, I always tried to provide a complete example on github, and reference to that, but that's often an overkill.

Thank you again!
Best regards,
Stefan

jfversluis commented:
Look into github gists. You can just paste the url of A public gist into Wordpress and it will show a fancy ‘hosted by github with <3’ box without needing a plug-in. Furthermore I totally agree with Cheesebaron. Keep up the good work!
